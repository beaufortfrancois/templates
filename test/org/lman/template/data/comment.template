{{- single line comment -}}
TODO make these render like "inline comment" not "inline  comment"
inline {{- hello -}} comment
{{- hello -}} leading comment
trailing comment {{- comment -}}
block comment:
{{- something here
    something else -}}
indented block comment:
  {{- something here
      something else -}}
crazy block {{-
I am a
multi-line
comment
-}}
not crazy anymore.
a bunch of single lines:
{{- here -}}
  {{- here -}}
    {{- here -}}
{{- here -}}
just an indented single line
  {{- here -}}
don't render the control characters
{{- Here are some {{control.characters}} inside an indented comment -}}
{{- Here are some {{control.characters}} inside a 
    block comment comment -}}
{{control.characters}}
unicorn
{{- and a {{- nested {{- nested -}} comment -}} and {{control.characters}}
and another {{- nested comment -}} and finally -}}
end
