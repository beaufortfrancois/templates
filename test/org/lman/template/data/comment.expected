foo  bar
baz 
quux 
seaslug
unicorn
